### DISCUSSION TITLE
Is there a documentation to visualize the model graph with Tensorboard or netron?

### DISCUSSION THREAD

**Discussion by chihyuwang** [facebookresearch/detectron2#3017] [General]
I am looking for a way to visualize the model graph in the detetron2 model zoo. Is there a way to integrate either Tensorboard or netron with detectron2?

**Comment by trongan93** [comment]
Check this one. Run in Jupyter.

```
%load_ext tensorboard
%tensorboard --logdir [path to trained output] --host [your jupyter IP]
```

**Reply by chihyuwang** [reply]
![image](https://user-images.githubusercontent.com/25482742/117595902-950e5800-b174-11eb-89a1-5b94d4c8d318.png)

I don't see any model graph in the tensorboard

**Reply by chihyuwang** [reply]
could you tell me where I could find the model graph in the tensorboard
